docs/management/connectors/action-types/slack.asciidoc

```diff
@@ -68,17 +91,21 @@ Message::   The message	text, converted to the `text` field in the Webhook JSON
 Use the <<action-settings,Action configuration settings>> to customize connector networking configurations, such as proxies, certificates, or TLS settings. You can set configurations that apply to all your connectors or use `xpack.actions.customHostSettings` to set per-host configurations.
 
 [float]
-[[configuring-slack]]
-==== Configure a Slack account
-
-You need a https://api.slack.com/incoming-webhooks[Slack webhook URL] to 
-configure a Slack account. To create a webhook
-URL, set up an an **Incoming Webhook Integration** through the Slack console:
+[[configuring-slack-webhook]]
```

Comment:
> [[configuring-slack-webhook]]

I believe the current doc build issue is related to the fact that this PR removes the `[[configuring-slack]]` anchor, which is used in https://github.com/elastic/kibana/blob/9517d067b5d7bb57d89bd62a1f786fda308da26b/packages/kbn-doc-links/src/get_doc_links.ts#L497, which in turn is used here: https://github.com/elastic/kibana/blob/9517d067b5d7bb57d89bd62a1f786fda308da26b/x-pack/plugins/stack_connectors/public/connector_types/slack/slack_webhook_connectors.tsx#L24

Reply:
I've re-added the `configuring-slack` anchor and made some other minor edits via 65c26c5e4b9c02948929f0be912a62cfff808856